### ISSUE TITLE
ether not showing in my wallet

### ISSUE THREAD

**Issue by sfsbitcoin** [ethereum/mist#3406]
Downloaded the Ethereum Wallet nearly 12 hour ago, version 0.9.3 Mac Version but ETH not showing in my wallet

SENDER | roshinshiroe007@gmail.com
-- | --
INPUT TRANSACTION HASH | 7c752a7e1f5489973264f4e830159cad3f13019fb1b7c296248e72c60608ff05
AMOUNT | 0.00184835 BTC
MONEY RECEIVED | 10 Dec 2017, 13:02:03
FEE | 0.00112499808160804 ETH
EXCHANGE RATE | 1 BTC = 30.38136457 ETH
RECEIVER | 0x033AB95e11A70dD14c0c5bAB77aEf57135dAFE78
OUTPUT TRANSACTION HASH | 0xe78111f4afeb10ee1730a8846fd426db805a651887b95fad45788c8a8460c8fc
AMOUNT | 0.05587461824 ETH
MONEY SENT | 10 Dec 2017, 13:31:36

 Plz Check This Link My Transaction
https://etherscan.io/tx/0xe78111f4afeb10ee1730a8846fd426db805a651887b95fad45788c8a8460c8fc

https://etherscan.io/address/0x033ab95e11a70dd14c0c5bab77aef57135dafe78

**Comment by ja72topu** [comment]
I am in the same situation.  :(

**Comment by Bryantng** [comment]
same! any resolution?

**Comment by Tolya9980** [comment]
same... and what should i do to get my money ??

**Comment by Tolya9980** [comment]
https://etherscan.io/tx/0x069a8952ddc6c7a450acf3528f9ab15254cdea0ceac3c692b8b35d1b0174edf9
almost a day passed, and the balance is still 0 eth

**Comment by ja72topu** [comment]
hey guys, I have an advice, if you don't want to wait synchronizing and down 30G data for the all blocks:
you can try MetaMask , a chrome web application. just upload the wallet file and it's easy and convenient.

**Comment by Tolya9980** [comment]
but there will be another purse and another address, no?

**Comment by ja72topu** [comment]
@Tolya9980 no, when you upload yourself's wallet file, you use the old address.

**Comment by Tolya9980** [comment]
and how to uploud this wallet file?

**Comment by Tolya9980** [comment]
so you have solved this problem as the guy above?

**Comment by ja72topu** [comment]
@Tolya9980 you will find it in the app. yes i have answered this question in another site.

**Comment by Tolya9980** [comment]
you mean select import account - json file ... yea?

**Comment by ja72topu** [comment]
@Tolya9980 right

**Comment by Tolya9980** [comment]
and what file do I need to insert?  it doesn't work((

**Comment by Tolya9980** [comment]
and i need to enter password from metamask or my old eth wallet?

**Comment by Tolya9980** [comment]
sory, but im stupid ))0))0
